test/integration/authenticationAPITests.js

```diff
@@ -20,6 +20,58 @@ import { config } from '../../src/config'
 const { SERVER_PORTS } = constants
 
 describe('API Integration Tests', () => {
+  describe('Retrieve Enabled Authentication types', () => {
+    let authDetails = null
+
+    before(async () => {
+      await testUtils.setupTestUsers()
+      authDetails = testUtils.getAuthDetails()
+      const startPromise = promisify(server.start)
+      await startPromise({ apiPort: SERVER_PORTS.apiPort })
+      await testUtils.setImmediatePromise()
+      await AuditModel.deleteMany({})
+    })
+
+    afterEach(async () => {
+      await AuditModel.deleteMany({})
+    })
+
+    after(async () => {
+      await Promise.all([
+        testUtils.cleanupTestUsers(),
+        promisify(server.stop)()
+      ])
+    })
+
```

Comment:
@bradsawadye Please add a test case for when the no authentication is configured.

Reply:
addressed in https://github.com/jembi/openhim-core-js/pull/1082/commits/562dc5c4c469b99a04afcc0d09a9d002436aac15